test/integrations/sources/cordial/data.ts

```diff
@@ -0,0 +1,626 @@
+import utils from '../../../../src/v0/util';
+
+const defaultMockFns = () => {
+  jest.spyOn(utils, 'generateUUID').mockReturnValue('97fcd7b2-cc24-47d7-b776-057b7b199513');
+};
+
+export const data = [
```

Comment:
Batched payload testcase is missing

Reply:
@Gauravudia  can you please check the 2nd testcase ? I have added 2 events and batched it.

Reply:
This is wrong. 
ref this payload :- https://www.notion.so/rudderstacks/Cordial-Source-3b5643f02d4245f88447d5c16b2dfd7f?pvs=4#82e7100960254a1cb99fe68b23e34f93

Reply:
done

Reply:
Testcase is still wrong. 
Cordial is sending batched event in this format 
`[{contact:{}, event:{}}]`

Reply:
What if some events in batch failed. I don't see any testcase for that.